DDD.Core/AppInsights/AppInsightsUser.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
 using System.Collections.Generic;
-using Microsoft.WindowsAzure.Storage.Table;
+using Microsoft.Azure.Cosmos.Table;
```

Comment:
I assume they indicate that it's OK to use this against a storage account rather than cosmos? My understanding is the Cosmos table API is compatible with table storage so I assume it's OK. We'll need to do a few solid tests in the test environment after merging this.

Reply:
Yes, you are right. According to MS the `Microsoft.WindowsAzure.Storage.Table` has been split into multiple parts. For table support, we should use the `Microsoft.Azure.Cosmos.Table` [Link](https://www.nuget.org/packages/WindowsAzure.Storage/)